themes/gatsby-theme-blog/src/pages/index.js

```diff
@@ -17,35 +16,36 @@ const BlogIndex = ({
   },
 }) => (
   <Layout location={location} title={siteTitle}>
-    <Bio />
-    {posts.map(({ node }) => {
-      const title = node.frontmatter.title || node.fields.slug
-      const keywords = node.frontmatter.keywords || [``]
-      return (
-        <Fragment key={node.fields.slug}>
-          <SEO title="Home" keywords={keywords} />
-          <div>
-            <Styled.h2
-              css={css({
-                mb: 1,
-              })}
-            >
-              <Styled.a
-                as={Link}
-                css={{
-                  textDecoration: `none`,
-                }}
-                to={node.fields.slug}
+    <Main as="main">
```

Comment:
The `as` prop shouldn't be necessary here after your Theme UI PR

Reply:
```suggestion
    <Main>
```

Reply:
cool cool -- what version should i bump to?

Reply:
v0.1.5 should have it but it should generally be okay to keep the theme-ui packages on latest as we're working on these